src/bundle/Resources/public/scss/ui/modules/muti-file-upload/_progress.bar.scss

```diff
@@ -1,38 +1,35 @@
 .c-progress-bar {
-    background: #d8d8d8;
-    height: calculateRem(6px);
     width: 100%;
-    border-radius: calculateRem(4px);
-    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
 
     &__value {
-        height: 100%;
-        background: $ibexa-color-primary;
+        background: linear-gradient(90deg, $ibexa-color-warning 0%, $ibexa-color-accent 44.95%, $ibexa-color-success 100%);
         border-radius: calculateRem(4px);
         transition: width 0.2s linear;
-    }
+        height: calculateRem(8px);
+        width: 20vh;
+        position: relative;
 
-    &__label,
-    &__uploaded {
-        position: absolute;
-        top: 0;
-        left: 50%;
-        font-weight: bold;
-        font-size: calculateRem(12px);
-        padding: 0 calculateRem(6px);
-        line-height: 1;
-        text-align: center;
-        width: 100%;
+        &::after {
+            content: '';
+            width: calc(100% - var(--progress));
+            height: calculateRem(10px);
+            border-radius: calculateRem(4px);
```

Comment:
Are you sure it should be 4px? Not 5px, which is variable in our system?

Reply:
Yes, because it makes the bar round but not too round ;)